packages/rest/src/__tests__/acceptance/caching-interceptor/global-caching-interceptor.acceptance.ts

```diff
@@ -0,0 +1,122 @@
+// Copyright IBM Corp. 2019. All Rights Reserved.
+// Node module: @loopback/rest
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {asInterceptor} from '@loopback/context';
+import {anOperationSpec} from '@loopback/openapi-spec-builder';
+import {get, param} from '@loopback/openapi-v3';
+import {
+  Client,
+  createRestAppClient,
+  expect,
+  givenHttpServerConfig,
+} from '@loopback/testlab';
+import {RestApplication} from '../../..';
+import {
+  cachedResults,
+  CachingInterceptorProvider,
+  clearCache,
+  status,
+} from './caching-interceptor';
+
+describe('global caching interceptor', () => {
+  let client: Client;
+  let app: RestApplication;
+
+  before(givenAClient);
+  after(async () => {
+    await app.stop();
+  });
+
+  context('toUpperCase', () => {
```

Comment:
This is confusing. IIUC, this block is verifying that global interceptors are applied on controller methods.

Please rework `context` name to capture this information and move the definition of the controller from the bottom of the test file into this particular `context` block that's using it.